lib/browser/api/web-contents.js

```diff
@@ -65,32 +65,33 @@ const PDFPageSizes = {
 
 // Default printing setting
 const defaultPrintingSetting = {
-  pageRage: [],
+  // Customizable.
+  pageRange: [],
   mediaSize: {},
   landscape: false,
-  color: 2,
   headerFooterEnabled: false,
   marginsType: 0,
-  isFirstRequest: false,
-  previewUIID: 0,
-  previewModifiable: true,
-  printToPDF: true,
+  scaleFactor: 100,
+  shouldPrintBackgrounds: false,
+  shouldPrintSelectionOnly: false,
+  // Non-customizable.
   printWithCloudPrint: false,
   printWithPrivet: false,
   printWithExtension: false,
   pagesPerSheet: 1,
+  isFirstRequest: false,
+  previewUIID: 0,
+  previewModifiable: true,
+  printToPDF: true,
   deviceName: 'Save as PDF',
   generateDraftData: true,
-  fitToPageEnabled: false,
-  scaleFactor: 100,
   dpiHorizontal: 72,
   dpiVertical: 72,
   rasterizePDF: false,
   duplex: 0,
   copies: 1,
-  collate: true,
-  shouldPrintBackgrounds: false,
-  shouldPrintSelectionOnly: false
+  color: 2,
```

Comment:
> i didn't actually change this in the scope of this PR

Ah I missed that part!

It would still be nice-to-have but I agree it's out-of-scope for the PR